javacutil/src/main/java/org/checkerframework/javacutil/ElementUtils.java

```diff
@@ -28,6 +29,7 @@
 import javax.lang.model.type.TypeVariable;
 import javax.lang.model.util.ElementFilter;
 import javax.lang.model.util.Elements;
+import javax.tools.JavaFileObject.Kind;
```

Comment:
I would have found the code easier if this just imported `JavaFileObject`, so that the condition reads `== JavaFileObject.Kind.SOURCE`. There are so many different `Kind` classes, so I would prefer not to import this.